app/templates/events/view/scheduler.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui grid">
   <div class="row">
-    <div class="three wide column">
+    <div class="{{if device.isMobile 'sixteen' 'three'}} wide column">
```

Comment:
Couldn't you make this 

`sixteen wide mobile three wide computer column` ?

Reply:
Completely forgot this thanks